frontend/test/metabase/scenarios/visualizations/drillthroughs/chart_drill.cy.spec.js

```diff
@@ -329,3 +360,9 @@ describe("scenarios > visualizations > drillthroughs > chart drill", () => {
     });
   });
 });
+
+function clickLineDot({ index } = {}) {
```

Comment:
I'm guessing that you are using click to position the mouse to show the tooltip, since there's not drill-through clicks on SQL

Reply:
Yes. And there is no hover in Cypress.